package.json

```diff
@@ -22,6 +22,7 @@
     "lint"
   ],
   "dependencies": {
+    "@getstation/electron-google-oauth2": "2.1.0",
```

Comment:
Note: I prefer to use a specific version when I use an external package for authentication.